src/decorators/getValidators.integration.test.ts

```diff
@@ -0,0 +1,75 @@
+import { describe, it, expect } from 'vitest';
+import { Address, createPublicClient, http } from 'viem';
+import { generatePrivateKey, privateKeyToAccount } from 'viem/accounts';
+
+import { nitroTestnodeL2 } from '../chains';
+import { rollupAdminLogicPublicActions } from './rollupAdminLogicPublicActions';
+import { getInformationFromTestnode, getNitroTestnodePrivateKeyAccounts } from '../testHelpers';
+import { getValidators } from '../getValidators';
+
+const { l3RollupOwner } = getNitroTestnodePrivateKeyAccounts();
+const { l3Rollup, l3UpgradeExecutor } = getInformationFromTestnode();
+
+const client = createPublicClient({
+  chain: nitroTestnodeL2,
+  transport: http(),
+}).extend(
+  rollupAdminLogicPublicActions({
+    rollup: l3Rollup,
+  }),
+);
+
+async function toggleValidator(validator: Address, state: boolean) {
```

Comment:
this function takes a boolean for state, so it isn't toggling an internal state

a better name should be `setValidator`?